app/views/sessions/updatesession.scala.html

```diff
@@ -97,111 +77,312 @@ <h2 style="margin-top:10px;">Update Session</h2>
                         </select>
                     </div>
                     <div class="col-md-2"><i class="fa fa-eye preview" id="previewId"></i></div>
-                    <div class="col-md-1"></div>
                 </div>
 
-                <div class="col-md-12 login-third update-second">
-                    <div class="col-md-1" ></div>
+                <div class="col-md-12 login-third update-second form-group">
+                    <label class="col-md-2 control-label">Topic</label>
                     <div class="col-md-10 " >
                         <input type="text" name="topic" id="topic" value='@sessionInfo.data.get("topic")' placeholder="Topic" required class="update-field"/>
                     </div>
-                    <div class="col-md-1" ></div>
                 </div>
 
-                <div class="col-md-12 ">
-                    <div class="col-md-1"></div>
-                    <div class="col-md-10 feedback-duration">
-                        <div class="col-md-12 duration-title">Feedback form expiration duration</div>
-                        <div class="col-md-6 feedback-expiration">
+                <div class="col-md-12 form-group">
+                    <label class="col-md-2 control-label">Feedback form expiration duration</label>
+                    <div class="col-md-10 feedback-duration ">
+                        <div class="col-md-4 feedback-expiration">
                             <label class="radio-button default-radio">
-                                <input type='radio' name='feedbackExpirationDays' id='disableDays' class='custom-checkbox' value='0' checked="checked"/>
+                                <input type='radio' name='feedbackExpirationDays' onclick='disableDays()' id='' class='custom-checkbox' value='0' checked="checked"/>
```

Comment:
Cross check if these changes are required. If yes, use event listeners instead of onclick.